src/ios/view.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * test view
+ *
+ * Created by Appcelerator, Inc
+ * Copyright (c) 2016 Appcelerator. All rights reserved.
+ */
+import { UIScreen } from 'UIKit/UIScreen';
+import { UIColor } from 'UIKit/UIColor';
+import { UIView } from 'UIKit/UIView';
+
+export default class View {
```

Comment:
so remove default? the problem i see in pragmatic terms if then the module key is View not default and you have to do some JS foo to determine that.